app/controllers/services_controller.rb

```diff
@@ -55,14 +55,15 @@ def other_country_message
 
   def hidden_user_explanation
     info_request = InfoRequest.find(params[:info_request_id])
-    render :template => "admin_request/hidden_user_explanation",
-      :content_type => "text/plain",
-      :layout => false,
-      :locals => {:name_to => info_request.user_name,
-                  :name_from => AlaveteliConfiguration.contact_name,
-                  :info_request => info_request, :reason => params[:reason],
-                  :info_request_url => 'http://' + AlaveteliConfiguration.domain + request_path(info_request),
-                  :site_name => site_name}
+    render template: "admin_request/hidden_user_explanation",
+           formats: [:text],
```

Comment:
Why has this changed?

Reply:
Gah, another missed commit comment _tracks down additional detail_

Without it, it defaults to `:formats=>[:html]` and doesn't find the renamed text template (raising an `ActionView::MissingTemplate` error). Prior to renaming, it rendered it as HTML but would have produced plain text output when the original code was written (for Rails 2.3) .

The `formats` option is described in  [the current documentation](https://guides.rubyonrails.org/layouts_and_rendering.html#the-formats-option) but isn't mentioned for Rails 4.2, despite apparently having been there from Rails 4.0. Before then, `content_type` would have done the trick as the format took its cue from the MIME type - https://github.com/rails/rails/commit/67b2404cf9b2c7e0f86bf0294571ef97391a6dcd

[This space left blank while I try to work out whether I need to set them both]